statistics/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt

```diff
@@ -84,16 +88,20 @@ interface Pixel {
         const val FIRE_ANIMATION_NONE = "fann"
     }
 
+    enum class PixelType { DEFAULT, DAILY, UNIQUE }
```

Comment:
I think it would be good to document here the differences between each PixelType